test/vagrant-service-manager/binary_handlers/adb_openshift_binary_handler_test.rb

```diff
@@ -0,0 +1,95 @@
+require 'test_helper'
+require 'vagrant/util/downloader'
+
+# Tests through ADBOpenshiftBinaryHandler to BinaryHandler
+module VagrantPlugins
+  module ServiceManager
```

Comment:
These tests are also only working on Linux. They fail on OS X, since they try to download OS X binaries. Even worse, because the right file cannot be downloaded system exit is called somewhere aborting the whole test run.